torch/csrc/Module.cpp

```diff
@@ -620,7 +620,7 @@ __declspec(dllexport)
 #endif
 PyObject* initModule() {
   HANDLE_TH_ERRORS
-  at::init_num_threads();
+  at::internal::lazy_init_num_threads();
```

Comment:
probably no need to do it here?

Reply:
This sets the thread local `init` variable as well. Providing a (marginal) speedup to the next `parallel_for` call.